modules/plottinggl/src/processors/parallelcoordinates/parallelcoordinates.cpp

```diff
@@ -342,6 +342,7 @@ void ParallelCoordinates::createOrUpdateProperties() {
     if (!data->getNumberOfRows()) return;
     if (!data->getNumberOfColumns()) return;
 
+    updating_ = true;
```

Comment:
use util::KeepTrueWhileInScope  (see my same fix in https://github.com/inviwo/inviwo/commit/160dcf73ea35d0aa847cc814bc3f3ef3cdffc69f)

Reply:
there is some code at the end of the function which requires the bool to be reverted to false again. When using KeepTrueWhileInScope one has to do that anyway or introduce another `{ }` block just for the raii thingy.

Reply:
I see